src/libraries/System.Management/src/System/Management/ManagementScope.cs

```diff
@@ -1178,11 +1178,8 @@ internal int QueryObjectSink_(int lFlags, ref IWbemObjectSink ppResponseHandler)
         }
         internal int GetObject_(string strObjectPath, int lFlags, IWbemContext pCtx, ref IWbemClassObjectFreeThreaded ppObject, IntPtr ppCallResult)
         {
-            //It is assumed that caller always passes ppCallResult as IntPtr.Zero.
-            //If it changes let this call go through wminet_utils.dll. Check implementation of CreateInstanceEnum_ for more information.
             int status = (int)tag_WBEMSTATUS.WBEM_E_FAILED;
-            if (!object.ReferenceEquals(ppCallResult, IntPtr.Zero))
-                status = pWbemServiecsSecurityHelper.GetObject_(strObjectPath, lFlags, pCtx, out ppObject, ppCallResult);
+            status = pWbemServiecsSecurityHelper.GetObject_(strObjectPath, lFlags, pCtx, out ppObject, ppCallResult);
             return status;
```

Comment:
Can't this just be:
```C#
return pWbemServiecsSecurityHelper.GetObject_(strObjectPath, lFlags, pCtx, out ppObject, ppCallResult);
```
?

Reply:
Yeah, I had originally written that, then noticed all of the other methods in this class declare status as FAILED, unconditionally reassign it from the wrapped call, then return it.  So I went with the "when in Rome" style rules.

Reply:
No need to initialize `status` when it's declared, then.

Reply:
> No need to initialize status when it's declared, then.

... other than the aforementioned matching of existing style.